src/components/media-loader.js

```diff
@@ -356,6 +357,11 @@ AFRAME.registerComponent("media-loader", {
         this.showLoaderTimeout = setTimeout(this.showLoader, 100);
       }
 
+      //check if url is an anchor hash e.g. #Spawn_Point_1
+      if (src.charAt(0) === "#") {
+        src = this.data.src = `${window.location.origin}${window.location.pathname}${src}`;
```

Comment:
Odd to be writing to `this.data.src` because it's AFRAME component state, but we wouldn't really want to `setAttribute` either because that would cause another call to `update`. :thinking:

Reply:
Yea, this was unfortunately necessary because other components specifically refer to this component's `data.src`